test/components/Timeline/Event/EventsTest.js

```diff
@@ -0,0 +1,149 @@
+'use strict'
+import { expect } from 'chai'
+
+import createComponent from 'test/helpers/shallowRenderHelper'
+
+import {
+  Events,
+  ConnectedEvent,
+  Event,
+  DisplayEvent,
+  mapStateToEventProps
+} from 'components/Timeline/Event/Events'
+import ErrorMessage from 'components/elements/ErrorMessage'
+import LoadingMessage from 'components/elements/LoadingMessage'
+
+describe('Events', function () {
+  describe('Events functional component', function () {
+    const input = {
+      events: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10].map(id => ({id})),
+      ticketId: 2,
+      incidentId: null
+    }
+
+    const result = createComponent(Events, input)
+
+    it('Should return a div containing an array of Events', function () {
+      expect(result.type).to.equal('div')
+      expect(result.props.children[0].type).to.equal(ConnectedEvent)
```

Comment:
Is there a way we can test this without exporting a ConnectedEvent? Is it valuable to test? [⎁](tktktk)

Reply:
It's not possible to test it accurately without exporting the component (ConnectedEvent) to the best of my knowledge. It is valuable to test because I have often accidentally made changes in such a way that I was using the non-connected version of a component where I was intending to use the connected version; this test would catch that.